source/getting_started/on-vm.rst

```diff
@@ -69,95 +30,39 @@ Reference the :ref:`build-os-image` section in the Getting Started Guide, and sp
 Create CaaS Virtual Disk
 ------------------------
 
-To create a virtual disk containing the CaaS partitions, the requirements are a USB flash drive and at least 16GB free disk space. Go through the following instructions to create and set up CaaS partitions on a *qcow2* formatted virtual disk.
+.. note::
+        This section is for flashing *flashfiles* package, no action required if you choose caas.img.
 
-#. Create a 16GB empty disk image of *qcow2* type:
-
-    .. code-block:: bash
-
-        $ qemu-img create -f qcow2 ~/caas/android.qcow2 16G
-
-#. Plug a USB flash drive to the development host, and identify the vendor ID and product ID of that USB flash drive from the output of the ``lsusb`` command:
-
-    .. code-block:: bash
-
-        $ lsusb
-        ...
-        Bus 002 Device 002: ID 0781:5591 SanDisk Corp.
-        ...
-
-    In the previous example, **0781** and **5591** are the vendor ID and product ID of the target USB flash drive, respectively.
-
-#. 6.Identify the directory you mounted the USB flash drive on or if not mounted, mount the USB flash drive to a temporary directory. Unzip the content of the CaaS *flashfile* package to the flash drive, and unmount the USB flash drive after finish:
-
-    .. code-block:: bash
-
-        $ sudo mount /dev/sdc /mnt   ## if the USB disk is not auto-mounted
-        $ sudo unzip caas-flashfiles-eng.<user>.zip -d /mnt
-        $ umount /dev/sdc
-
-    .. note::
-        The previous example assumes you assigned /dev/sdc to the USB flash drive. You might need to replace the drive names with the actual device node observed from the ``lsblk`` command.
-
-#. Download the helper script ``start_flash_usb.sh`` and edit the script with the vendor ID and product ID for your USB flash drive observed in the previous step:
-
-    .. code-block:: bash
-
-        $ wget https://raw.githubusercontent.com/projectceladon/device-androidia-mixins/master/groups/device-specific/caas/start_flash_usb.sh  -O ~/caas/start_flash_usb.sh
-        $ chmod +x ~/caas/start_flash_usb.sh
-
-    .. code-block:: none
-
-        #!/bin/bash
-
-        qemu-system-x86_64 \
-        ...
-        -device usb-host,bus=xhci.0,vendorid=<your-usb-vendorid>,productid=<your-usb-productid> \
-        ...
-        -bios ./OVMF.fd \
-
-#. After launching the ``start_flash_usb.sh`` script, a QEMU window will pops up. Click the QEMU window, press the **F2** key, and select the **Boot Manager** option to enter the boot-manager menu:
+#. Run the helper script ``start_flash_usb.sh``.
 
     .. code-block:: bash
 
         $ cd ~/caas
-        $ sudo ./start_flash_usb.sh
-
-    .. figure:: images/qemu-bios.png
-        :align: center
+        $ sudo ./start_flash_usb.sh caas-flashfiles-eng.<user>.zip
```

Comment:
In the previous script, the user's USB disk is called out specifically by matching its VID & PID, is it sufficient to only pass below argument to the QEMU?
```
-device usb-storage,drive=udisk1,bus=xhci.0 \
```

Reply:
yes, we are using the emulation USB device to do this.